exercises/sublist/src/sublist.h

```diff
@@ -0,0 +1,16 @@
+#ifndef SUBLIST_H
+#define SUBLIST_H
+
+#include <stddef.h>
+
+enum {
+   EQUAL,
+   UNEQUAL,
+   SUBLIST,
+   SUPERLIST
+};
+
+int check_lists(int *listOne, int *listTwo, size_t listOneLength,
```

Comment:
The return value should be the enum type.

Reply:
👍